app/src/main/java/ch/epfl/sdp/blindly/BlindlyApplication.kt

```diff
@@ -0,0 +1,11 @@
+package ch.epfl.sdp.blindly
+
+import android.app.Application
+import android.content.res.Configuration
+import dagger.hilt.android.HiltAndroidApp
+
+// Custom app to let hilt hook into
+@HiltAndroidApp
+class BlindlyApplication : Application() {
+
+}
```

Comment:
Why is this empty class relevant ?

Reply:
Just read the comment above ;)

Reply:
Is this to inject manual dependencies?
I'm not sure I understand how this works... Couldn't we do without this empty class?